### PR TITLE
#500: Add NSP to lint command in CI.

### PR DESCRIPTION
**Pull request by Osmose**
Currently includes a commit that should cause nsp to fail CI, will remove once that's confirmed working.

### CODE REVIEW

File: lints/package.json [hunk 1]
```diff
@@ -31,6 +32,7 @@
     "redux": "3.6.0",
     "redux-form": "6.1.0",
     "redux-thunk": "2.1.0",
+    "serialize-to-js": "^0.5.0",
```
Comment (mythmon) [inline lints/package.json]: What's this? Is it a peer dependency that we have to install explicitly?

**Comment by mythmon** [comment]
Looks like nsp succesfully caught the issue in 25de33d.

File: lints/package.json (hunk 1 above)
Comment (Osmose) [inline lints/package.json]: It's a library with a known vulnerability that I'm using to test the CircleCI job.

Comment (mythmon) [inline lints/package.json]: Oh, derp. I didn't look at what the actually changes in the second commit were.

**Comment by Osmose** [comment]
Righto, this is ready for review!

**Pull request merged**